src/main/java/pl/javastart/task/io/InputReader.java

```diff
@@ -0,0 +1,17 @@
+package pl.javastart.task.io;
+
+import java.util.Scanner;
+
+public class InputReader {
+    private Scanner input = new Scanner(System.in);
+
+    public int getInt() {
+        int number = input.nextInt();
+        input.nextLine();
+        return number;
+    }
+
+    public void close() {
+        input.close();
+    }
```

Comment:
Nie ma potrzeby zamykania scannera, jeśli działa na System.in. Scanner może działać również na strumieniu z pliku, wtedy koniecznie zamykamy.